crates/tools/gnullvm/readme.md

```diff
@@ -0,0 +1,16 @@
+The Windows umbrella lib (targeting mingw-w64 LLVM tooling) is generated using the following steps:
+
+0. Ensure MSYS2 MinGW environment is installed (https://www.mingw-w64.org/downloads/)
+1. Ensure `~\.cargo\config` has the following block to override toolchain defaults (customize paths as needed):
+
+    ```toml
+    [target.x86_64-pc-windows-gnu]
+    linker = "C:\\msys64\\mingw64\\bin\\x86_64-w64-mingw32-gcc.exe"
+    ar = "C:\\msys64\\mingw64\\bin\\ar.exe"
+    ```
+
+2. Open `MSYS2 MinGW 64-bit`
+3. Execute: `pacman -Syuu --noconfirm` (repeat until no further updates available)
+4. Execute `pacman --needed -S mingw-w64-x86_64-llvm`
+5. Navigate to crate root
+6. Execute: `PATH=$USERPROFILE/.cargo/bin:$PATH cargo run -p tool_gnullvm --target x86_64-pc-windows-gnu -- all`
```

Comment:
Can we use clang directly? Would like to avoid piggy backing off the gnu target/gcc as much as possible. Any general pointers/tips on building your toolchain locally for testing?

Reply:
> Can we use clang directly?

There would be no problem in using Clang as the linker for `x86_64-pc-windows-gnu` target but it would still use all other GNU tools like `ld.bfd`, `libgcc`, etc.

To be honest it doesn't matter which toolchain we use here, it can be any target since we use it only to build `tool_gnullvm` so I based it off `tool_gnu`.
Tools called inside `tool_gnullvm` are unaffected by target that built it (with some changes one could even run it using Linux target).

Reply:
That's OK, just trying to use as little of that _older stuff_ as possible. 😄

I was able to run the tool with no problems with your new target + rustc nightly, so I'd go for that!